src/set_theory/cardinal/basic.lean

```diff
@@ -824,7 +824,9 @@ by rw [←lift_nat_cast.{v} n, lift_inj]
 
 theorem lift_mk_fin (n : ℕ) : lift (#(fin n)) = n := by simp
 
-lemma mk_finset {α : Type u} {s : finset α} : #s = ↑(finset.card s) := by simp
+lemma mk_of_finset {α : Type u} {s : finset α} : #s = ↑(finset.card s) := by simp
+
+lemma mk_finset_of_fintype [fintype α] : #(finset α) = 2 ^ℕ fintype.card α := by simp
```

Comment:
Can you squeeze this simp, for the sake of making it clear which lemmas this uses?

Reply:
This `simp` basically just turns `#(finset α)` into `fintype.card (finset α)`, for which the lemma stating that it's equal to `2 ^ fintype.card α` already exists. A few other proofs in this file, like `mk_fin`, `mk_bool`, `mk_Prop`, use this same technique and are not squeezed.

Should I squeeze it anyways?

Reply:
I think that non-squeezed `simp`s are OK as long as they're fast.